From 62639364a104859e693cfe3a52f1555db646c08e Mon Sep 17 00:00:00 2001
From: Gilles Dartiguelongue <eva@gentoo.org>
Date: Mon, 8 Mar 2010 22:49:04 +0100
Subject: [PATCH] Revert "fix build problems with libunique and -DGTK_DISABLE_SINGLE_INCLUDES compiler flag"

Gentoo: This is not a problem as we do not compile with deprecated code
directives.

This reverts commit e494be096231cc676cebf23cc4f28dddda6e8d1a.
---
 configure.ac |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 8aaddb1..cd956ea 100644
--- a/configure.ac
+++ b/configure.ac
@@ -51,7 +51,6 @@ CLUTTER_REQUIRED=1.0.0
 CLUTTER_GTK_REQUIRED=0.10.0
 GSTREAMER_REQUIRED=0.10
 OPENRAW_REQUIRED=0.0.8
-UNIQUE_REQUIRED=1.1.2
 LIBSOUP_REQUIRED=2.26
 GNOME_KEYRING_REQUIRED=2.28
 
@@ -120,7 +119,7 @@ PKG_CHECK_MODULES(GTHUMB, [
 	gio-unix-2.0
 	gtk+-2.0 		>= $GTK_REQUIRED
 	gconf-2.0 		>= $GCONF_REQUIRED
-	unique-1.0		>= $UNIQUE_REQUIRED
+	unique-1.0
 ])
 AC_SUBST(GTHUMB_LIBS)
 AC_SUBST(GTHUMB_CFLAGS)
-- 
1.7.0

